amundsen_application/static/js/components/common/ShimmeringTagListLoader/index.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import * as times from 'lodash/times';
+
+import './styles.scss';
+
+const DEFAULT_REPETITION = 10;
+
+type ShimmeringTagItemProps = {
+  index: number;
+};
+
+export const ShimmeringTagItem: React.SFC<ShimmeringTagItemProps> = ({
+  index,
+}: ShimmeringTagItemProps) => {
+  return (
+    <span
+      className={`shimmer-tag-loader-item shimmer-tag-loader-item--${index} is-shimmer-animated`}
+    />
+  );
+};
+
+export interface ShimmeringTagListLoaderProps {
+  numItems?: number; // Max items would be 15
```

Comment:
Is max items concept from design? If so we could also enforce that in the code with `Math.max()`.

Reply:
I actually tried to enforce it, but didn't know the best way to go about it.
Throwing an error didn't work as expected, any ideas of how to approach it?

Reply:
By "enforce" I don't think we need to go as far as making it such that the app throws an error but
1. What happens if we try `numItems?: 1 | 2 | 3 | ... | 15`?
2. If that doesn't do anything at best my original idea was to enforce a maximum of 15 in the render logic.

Reply:
Oh, that makes sense!
Thanks!